moped-editor/src/views/projects/projectView/ProjectFundingTable.js

```diff
@@ -299,11 +296,11 @@ const ProjectFundingTable = () => {
     }, {});
   };
 
-/**
- * Component for autocomplete using a lookup table as options
- * @param {*} props
- * @returns {React component}
- */
+  /**
+   * Component for autocomplete using a lookup table as options
+   * @param {*} props
+   * @returns {React component}
+   */
   const LookupAutocompleteComponent = (props) => (
```

Comment:
This could be refactored down the line but I agree that seeing these components move outside of the `ProjectFundingTable` component could be a good improvement. This is a totally a pattern that we have all used in this project, and I had a TIL moment that came from John and I chatting about anti-patterns the other day and me thinking through the pitfalls of components defined inside other components.

https://oozou.com/blog/6-react-anti-patterns-to-avoid-206

Reply:
ooh thanks for that blog post. I was wondering how the components defined inside other components render